src/main/scala/uk/gov/nationalarchives/tdr/validation/MetadataCriteria.scala

```diff
@@ -0,0 +1,47 @@
+package uk.gov.nationalarchives.tdr.validation
+
+case class Metadata(name: String, value: String)
+case class MetadataCriteria(
+    name: String,
+    dataType: DataType,
+    required: Boolean,
+    isFutureDateAllowed: Boolean,
+    isMultiValueAllowed: Boolean,
+    definedValues: List[String],
+    requiredProperty: Option[String] = None,
+    dependencies: Option[Map[String, List[MetadataCriteria]]] = None,
+    defaultValue: Option[String] = None
+)
+
+object MetadataProperty {
+  val closureType = "ClosureType"
+  val descriptiveType = "DescriptiveType"
+}
+
+object ErrorCode {
```

Comment:
`INVALID_DATE_FORMAT_ERROR` and `INVALID_NUMBER_ERROR` don't appear to be used any where in the code.

Is there some checks that are missing, or using the incorrect error, or are they simply not needed?

Reply:
Good catch. Added a check for `INVALID_DATE_FORMAT_ERROR` and `INVALID_NUMBER_ERROR` is not needed so I removed it